### ISSUE TITLE
add delete/reject buttons to pending questions & notebooks

### ISSUE THREAD

**Issue by nangchrys** [Metaculus/metaculus#913]
Previously when clicking "Approval" moderators had the option to delete or reject a question. This is not longer there and the only way to delete spam or bad questions is to bug the admins.

**Comment by SylvainChevalier** [comment]
Yes good idea.

**Comment by ryooan** [comment]
@SylvainChevalier I marked this as time sink, would be helpful to be able to do this from the front end. We also had a request from an admin of a private project for how to delete a question today, and would be nice to have this capability.

**Comment by SylvainChevalier** [comment]
@ryooan I notice there is also no way to send a question back to the review queue after it's approved.

So we need:

1. Add "Reject" and "Delete" buttons to review CTA, accessible to both admin and mods.
![Image](https://github.com/user-attachments/assets/e7d093fb-034c-464d-90c7-77ec606f6d49)
2. Add "Send back to review" and "Delete" buttons to ••• menu, only for admins.
![Image](https://github.com/user-attachments/assets/6be55ad4-a0dd-4ccf-859f-1d747164c82f)
3. Both the reject and delete buttons should bring a "Confirm {deletion,rejection}" modal with Yes/No buttons.

**Comment by skmmcj** [comment]
I think a Delete button for admins **and question authors**.

**Comment by ryooan** [comment]
Yeah for question authors, but only when the question is in draft or pending.

**Comment by SylvainChevalier** [comment]
@skmmcj @ryooan thank you both, I updated the comment above.
